LigatureLayoutBug/LigatureLayoutBug/ExampleCollectionView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ExamplesView.swift
+//  LigatureLayoutBug
+//
+//  Created by Nicole Souvenir on 10/31/18.
+//  Copyright © 2018 Nicole Souvenir. All rights reserved.
+//
+
+import UIKit
+
+class ExampleCollectionView: UICollectionView {
+    
+    let dataObject: [String]
+    
+    init(withDataObject dataObject: [String], minimumLineSpacing: CGFloat) {
+        
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.minimumLineSpacing = minimumLineSpacing //between items in the same grid
+        flowLayout.minimumInteritemSpacing = 8.0 //between items in the same row
+        flowLayout.sectionInset = .zero
+        flowLayout.scrollDirection = .horizontal
+        
+        self.dataObject = dataObject
+        super.init(frame: .zero, collectionViewLayout: flowLayout) //must have designated initializer
+        
+        self.dataSource = self
+        self.delegate = self
+        self.register(LabelCollectionViewCell.self, forCellWithReuseIdentifier: "ButtonCell")
+        self.backgroundColor = .blue
+        self.showsHorizontalScrollIndicator = false
+
+        self.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            self.heightAnchor.constraint(equalToConstant: flowLayout.itemSize.height),
+            self.topAnchor.constraint(equalTo: self.topAnchor),
+            self.leadingAnchor.constraint(equalTo: self.leadingAnchor),
+            self.trailingAnchor.constraint(equalTo: self.trailingAnchor)
+            ])
```

Comment:
This code is a bit confusing.  I see that you are trying to set the height of the collectionView but you are also just resetting all of the other anchors to themselves which should in theory have no effect. 

I think you could achieve your purpose with a:

`heightAnchor.constraint(equalToConstant: flowLayout.itemSize.height).isActive = true`